scripts/ci/postgres.sh

```diff
@@ -2,6 +2,9 @@
 
 # Adapted from https://github.com/stackrox/stackrox/blob/master/scripts/ci/jobs/go-postgres-tests.sh
 
+SCRIPTS_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")"/../.. && pwd)"
+source "$SCRIPTS_ROOT/scripts/lib.sh"
+
```

Comment:
Why this was added?

Reply:
Updated the description, but it's because we use `info` and `die` here, and those are defined in `scripts/lib.sh`. Got an error message from CI about it:

`info: No menu item 'Starting Postgres' in node '(dir)Top'`

Reply:
:+1: